airbyte-integrations/connectors/source-bing-ads/source_bing_ads/reports.py

```diff
@@ -337,3 +341,14 @@ def stream_slices(
             yield {"account_id": account["Id"], "customer_id": account["ParentCustomerId"]}
 
         yield from []
+
+
+class PerformanceReportsMixin(ReportsMixin):
+
+    def _get_start_date(self, stream_state: Mapping[str, Any] = None, account_id: str = None):
+        if not stream_state or not account_id or not stream_state.get(account_id, {}).get(self.cursor_field):
+            start_date = self.client.reports_start_date.subtract(days=self.config["lookback_window"])
```

Comment:
_Note_: attribution window on Bing Ads UI can be set at a minutes-granularity. For now, it is set to a days-granularity due to a bug encountered with `pendulum` library in the development process.

How to reproduce the bug:
```
import datetime

import pendulum
#version 2.1.2

dt = pendulum.datetime(2023, 1, 1).astimezone(tz=datetime.timezone.utc)

# works
dt.subtract(weeks=2)

#works
dt.subtract(days=2)

# doesn't work
dt.subtract(minutes=2)

# doesn't work
dt.subtract(seconds=2)
```

This is a known issue of `pendulum` (cf [here](https://github.com/sdispater/pendulum/issues/527)).